docs/history/changelog.md

```diff
@@ -35,6 +35,7 @@ Per the [normative and non-normative content and changes policy](https://docs.go
 
 ### Codelists
 
+* [#1200](https://github.com/open-contracting/standard/pull/1200) Deprecate all the codes of the `submissionMethod` codelist.
```

Comment:
Add an entry about deprecating the field as well.

Reply:
Done, I have also summarized why the codes are deprecated.